src/test/java/com/joss/conductor/mobile/LocomotiveTest.java

```diff
@@ -303,6 +303,68 @@ public void call() throws Throwable {
         }).isInstanceOf(IllegalArgumentException.class);
     }
 
+    @Test
+    public void test_perform_corner_swipe() {
+        WebDriver.Window window = mock(WebDriver.Window.class);
+        when(window.getSize()).thenReturn(new Dimension(100, 100));
```

Comment:
I think it would help to set the screen size to a more realistic example, 100x100 is pretty small.

Reply:
I agree for the most part, but to me this is meant to test functionality and is the pattern we've been using for most of our tests. I don't know that its a good idea to over complicate the test

Reply:
making my tests use an iPhone 6 plus

Reply:
changed to test against an iPhone 6 Plus with latest commit